### PR TITLE
Admin Areas 

### PR DESCRIPTION
**Pull request by jdtoombs**
Adding a number of administrative area management pages.

Wondering if I should maybe combine some of these models/services/filters or if they are going to be customized further in the future.

### CODE REVIEW

**Review by sturufous — APPROVED** [review]

File: app/editor/src/features/admin/actions/constants/defaultAction.ts [hunk 1]
```diff
@@ -0,0 +1,9 @@
+import { IMediaTypeModel } from 'hooks';
+
+export const defaultAction: IMediaTypeModel = {
```
Comment (Fosol) [inline app/editor/src/features/admin/actions/constants/defaultAction.ts:3]: Shouldn't this be a `IActionModel`?

File: app/editor/src/features/admin/categories/constants/defaultCategory.ts [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { IMediaTypeModel } from 'hooks';
+
+export const defaultCategory: IMediaTypeModel = {
```
Comment (Fosol) [inline app/editor/src/features/admin/categories/constants/defaultCategory.ts:3]: Shouldn't this be a `ICategoryModel`?

File: app/editor/src/features/admin/series/constants/defaultSeries.ts [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import { IMediaTypeModel } from 'hooks';
+
+export const defaultSeries: IMediaTypeModel = {
```
Comment (Fosol) [inline app/editor/src/features/admin/series/constants/defaultSeries.ts:3]: Shouldn't this be a `ISeriesModel`?

File: app/editor/src/features/admin/tags/constants/defaultTag.ts [hunk 4]
```diff
@@ -0,0 +1,9 @@
+import { IMediaTypeModel } from 'hooks';
+
+export const defaultTag: IMediaTypeModel = {
```
Comment (Fosol) [inline app/editor/src/features/admin/tags/constants/defaultTag.ts:3]: Shouldn't this be a `ITagModel`?

**Review by Fosol — CHANGES_REQUESTED** [review]

File: api/net/Areas/Admin/Controllers/CategoryController.cs [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    #endregion
+
+    #region Endpoints
+    /// <summary>
+    /// Find a page of category for the specified query filter.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("all")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(CategoryModel), (int)HttpStatusCode.OK)]
```
Comment (Fosol) [inline api/net/Areas/Admin/Controllers/CategoryController.cs:52]: This should be an array or `IEnumerable` as we don't return a single `CategoryModel`

File: api/net/Areas/Admin/Controllers/SeriesController.cs [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    #endregion
+
+    #region Endpoints
+    /// <summary>
+    /// Find a page of series for the specified query filter.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("all")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(SeriesModel), (int)HttpStatusCode.OK)]
```
Comment (Fosol) [inline api/net/Areas/Admin/Controllers/SeriesController.cs:52]: This should be an array or `IEnumerable` as we don't return a single `SeriesModel`

File: api/net/Areas/Admin/Controllers/TagController.cs [hunk 7]
```diff
@@ -0,0 +1,142 @@
...
+    #endregion
+
+    #region Endpoints
+    /// <summary>
+    /// Find a page of category for the specified query filter.
+    /// </summary>
+    /// <returns></returns>
+    [HttpGet("all")]
+    [Produces("application/json")]
+    [ProducesResponseType(typeof(IPaged<TagModel>), (int)HttpStatusCode.OK)]
```
Comment (Fosol) [inline api/net/Areas/Admin/Controllers/TagController.cs:52]: This should be an array or `IEnumerable` as we don't return a a page of `TagModel`

**Review by Fosol — CHANGES_REQUESTED** [review]

**Review by Fosol — APPROVED** [review]

**Pull request merged**